packages/edge-config/src/utils/tracing.ts

```diff
@@ -1,79 +1,91 @@
-import type { TracerProvider, Tracer, Attributes } from '@opentelemetry/api';
+import {
+  trace as traceApi,
+  type Tracer,
+  type Attributes,
+} from '@opentelemetry/api';
 import { name as pkgName, version } from '../../package.json';
 
-let tracerProvider: TracerProvider | undefined;
-
-export function setTracerProvider(nextTracerProvider: TracerProvider): void {
-  tracerProvider = nextTracerProvider;
+function getTracer(): Tracer | null {
```

Comment:
Nit: this API never returns `null`. But ok if you need it for some reason.

Reply:
Good catch, forgot to remove it from the previous implementation